environment.yml

```diff
@@ -17,11 +17,15 @@ dependencies:
 - pulp
 - paramtools
 - pypandoc
+- openpyxl
```

Comment:
@bodiyang, Are these 4 new dependencies used anywhere in the `taxdata` package?  e.g., I like `plotly`, but if `taxdata` doesn't call it, we don't want to include it as a dependency here.

Reply:
@jdebacker it is required in generating report in history/report.py